apps/nextjs-website/package.json

```diff
@@ -15,31 +15,33 @@
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
     "@markdoc/markdoc": "0.3.0",
-    "@mui/icons-material": "^5.11.0",
-    "@mui/lab": "5.0.0-alpha.119",
-    "@mui/utils": "5.12.3",
-    "@mui/material": "5.11.8",
-    "@pagopa/mui-italia": "0.9.5",
-    "@pagopa/pagopa-editorial-components": "2.0.0",
-    "@stoplight/elements": "^7.7.18",
+    "@mui/icons-material": "^5.14.3",
+    "@mui/lab": "^5.0.0-alpha.140",
+    "@mui/utils": "^5.14.5",
+    "@mui/material": "^5.14.5",
+    "@pagopa/mui-italia": "1.0.1",
+    "@pagopa/pagopa-editorial-components": "2.1.3",
+    "@stoplight/elements": "^7.9.0",
     "fp-ts": "^2.13.1",
     "gitbook-docs": "*",
-    "next": "^13.4.6",
+    "next": "^13.4.19",
```

Comment:
You could also try version 13.5. [Looking at the changelog](https://nextjs.org/blog/next-13-5), it could help us to solve some issues we have.

Reply:
The upgrade to the new version of Next is in this PR: #268